sdk/nextjs/src/common/invalidateConfig.ts

```diff
@@ -0,0 +1,9 @@
+'use server'
+import { invalidateConfigCache } from '../server/requests'
+
+export const invalidateConfig = async (
+    sdkToken: string,
+    lastModified?: number,
+): Promise<void> => {
+    await invalidateConfigCache(sdkToken, lastModified)
```

Comment:
whats the reason for wrapping this method and passing the same params?

Reply:
this one is a server action which wraps an internal method that can be used by other parts of the SDK